wiki/React-Best-Practices-And-Improvements.md

```diff
@@ -1,78 +1,144 @@
+_Certified fresh 230815 by Graham Johnson. All React recommendations originally authored 230502 by Marcin Maslanka._
+
+- ⚠️ This entry intentionally stays silent on state management. See [State Management](./State-Management.md) entry instead.
+- ⚠️ This entry is not oriented toward performance improvements. It is concerned with improving developer experience, increasing readability, and aligning our codebase with current React standards.
+- ⚠️ This entry discusses aspirational ideals for our codebase, and is not a descriptive documentation of how our codebase is actually organized. Strong divergences between ideal and actual have been explicitly flagged, but few or none of the practices  delineated are consistently or fully implemented in our present codebase (230815).
```

Comment:
good point! I have attempted to start refactoring to adjust codebase to the new standards in these PRs https://github.com/hicommonwealth/commonwealth/pull/3815 and https://github.com/hicommonwealth/commonwealth/pull/3839 but unfortunately there was "more important" work to do so I stopped working on https://github.com/hicommonwealth/commonwealth/issues/3808

We should try to gradually move away from old to new standards and we should always follow new standards when we create new features & files